packages/@ember/application/instance.ts

```diff
@@ -271,36 +299,32 @@ const ApplicationInstance = EngineInstance.extend({
     return router
       .handleURL(location.getURL())
       .then(handleTransitionResolve, handleTransitionReject);
-  },
+  }
 
   willDestroy() {
```

Comment:
Do we expect people will/should override this?

Reply:
@rwjblue, thoughts?